cookbook/email/gmail.rst

```diff
@@ -75,3 +75,9 @@ You're done!
 
     The ``gmail`` transport is simply a shortcut that uses the ``smtp`` transport
     and sets ``encryption``, ``auth_mode`` and ``host`` to work with Gmail.
+
+.. note::
+
+    Depending on your Gmail account settings, you may get authentication errors within your app.
+    You should ensure two-factor authentication and [also allow less secure apps to access your
+    account](https://support.google.com/accounts/answer/6010255)
```

Comment:
this is markdown syntax, while this docs is writting in reStructured Text. Please update this PR to use that syntax:

```rst
.. note::

    Depending on your Gmail account settings, you may get authentication errors within your app.
    You should ensure two-factor authentication and 
    `also allow less secure apps to access your account`_.

.. _`also allow less secure apps to access your account`: https://support.google.com/accounts/answer/6010255
```